### PR TITLE
add sensor_* variables description

### PR DESCRIPTION
**Pull request by mbernasocchi**
Relates to https://github.com/opengisch/QField/pull/1608/

### CODE REVIEW

**Comment by m-kuhn** [comment]
I guess for most cases the `@position_xxx` variables are more appropriate than the sensor ones, I think these should be the most prominent ones.

**Review by m-kuhn — CHANGES_REQUESTED** [review]
Use `position_xxx` as the main documentation

File: en/prepare/attributes-form.rst [hunk 1]
```diff
@@ -217,75 +217,76 @@ expression variables. These will only be available when the positioning
...
 
-All @sensor_xxx variables have a corresponding @position_xxx variable. They have the same value unless the point is manually set in which case the value is `NULL`.
-For example:
- - `@position_horizontal_accuracy`
-   - The horizontal accuracy of the coordinate (in meters) as reported by the sensor. If the point is manually set the value is `NULL`.
-   - IE
- - `@position_vertical_accuracy`
-   - The vertical accuracy of the coordinate (in meters) as reported by the sensor. If the point is manually set the value is `NULL`.
-   - IE
+All @position_* variables have a corresponding @gnss_* variable. The sensor variables always report the gnss sensor values.
```
Comment (m-kuhn) [inline en/prepare/attributes-form.rst:278]:
```suggestion
All @position_* variables have a corresponding @gnss_* variable. The gnss variables always report the gnss sensor values, even when the crosshair is not snapped.
```

**Review by m-kuhn — APPROVED** [review]
good work

File: en/prepare/attributes-form.rst [hunk 2]
```diff
@@ -218,65 +218,77 @@ is enabled, either through the internal device receiver or through an
...
-   - The GPS Fix Status "NoData", "NoFix", "Fix2D" or "Fix3D"
+   - The GPS Fix Status "NoData", "NoFix", "Fix2D" or "Fix3D" as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
    - E
  - `@position_fix_mode`
-   - Fix mode (where 'M' = Manual, forced to operate in 2D or 3D or 'A' = Automatic, 3D/2D)
+   - Fix mode (where 'M' = Manual, forced to operate in 2D or 3D or 'A' = Automatic, 3D/2D) as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
    - E
 
 
+All @position_* variables have a corresponding @gnss_* variable. The gnss variables always report the gnss sensor values, even when the crosshair is not snapped.
```
Comment (LindaOPENGIS) [inline en/prepare/attributes-form.rst:278]:
```suggestion
All @position_* variables have a corresponding @gnss_* variable. The @gnss_* variables always report the gnss sensor values, even when the crosshair is not snapped.
```

**Review by LindaOPENGIS — APPROVED** [review]

**Comment by m-kuhn** [comment]
Sorry, I was a bit too fast with merging, you can open a new pull request

**Comment by mbernasocchi** [comment]
no worries, I was showing Linda how to insert suggestions :)

**Pull request merged**